README.md

```diff
@@ -20,9 +20,34 @@ Community-maintained package to support using [Rust](https://www.rust-lang.org/)
 
 The below documentation is for the `vercel_runtime` crate (in beta). If you are looking for the legacy runtime instructions using `vercel_lambda` see [tree/a9495a0](https://github.com/vercel-community/rust/tree/a9495a0f0d882a36ea165f1629fcc79c30bc3108).
 
+## Introduction
+
+This section aims to give you a basic understanding about Vercel runtimes.
+
+### Builder module
+
+The _npm_ module `vercel-rust` is implementing an interface which is primarily taking care of spawning a development server, caching between consecutive builds, and running the compilation. You can read more about the in-depths of implementing a builder [here](https://github.com/vercel/vercel/blob/main/DEVELOPING_A_RUNTIME.md).
+
+Note that this dependency **does not** have to be installed manually, as it is pulled automatically by the Vercel CLI or the build container during deployments.
```

Comment:
IMO, this is leaking implementation details, in a funny kinda way. 

Should we document this under `contributing` instead of the README?

Reply:
Issue being that people seem to be installing the dependency even though they do not need to. We could make the description more generic with less information?

Reply:
I think if the dependency here isn't mentioned, it should be okay? It's a bit of a red herring for unclear step-by-step instructions to get started anyway I think.

Reply:
It is mentioned in the `functions` property in `vercel.json`. I think I will move this into `Contributing` and add a simple note in the usage section that it does not have to be installed.